src/main/java/com/ivoyant/kafkaemailevent/config/KafkaConfig.java

```diff
@@ -11,4 +11,9 @@ public NewTopic createTopic() {
         return new NewTopic("email-event", 3, (short) 1);
```

Comment:
Topic name should come from properties file.

Reply:
okay i will change and raise new pr